webpack/ForemanRhCloudHelpers.js

```diff
@@ -4,3 +4,7 @@
  * should be imported once core moves it to the ReactApp folder.
  */
 export const foremanUrl = path => `${window.URL_PREFIX}${path}`;
+
+export const isNotRhelHost = ({ hostDetails }) =>
+  // eslint-disable-next-line camelcase
+  !new RegExp('red\\s?hat', 'i').test(hostDetails?.operatingsystem_name);
```

Comment:
'operatingsystem_name' is not safe as this value can be changed by the user. As this plugin depends on Katello anyways, you can savely use `hostDetails?.operatingsystem_family`

Reply:
Well, you will get the OS type Redhat for Alma, Rocky, etc. as well, so this is tricky... But maybe you can use it for a sanity check.

Reply:
There is no better check for this right now. I would just go with this solution.

Reply:
What about checking at least for `RHEL` ?  I don't know how people are naming their OSes these days.

Reply:
on Redhat, these OS are created automatically from the imported Manifest. 
Other areas will not work if OS name was changed, like https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/snippet/insights.erb#L10